source/advocating.html.erb.md

```diff
@@ -11,41 +11,41 @@ status: editors-draft
 [ ... Tip description goes here ... ]
 
 {:.attach_permalink}
-## Develop accessibility business case
+## Develop an accessibility business case
 
 [ ... Tip description goes here ... ]
 
 {:.attach_permalink}
-## Secure management support for accessibility activities
+## Address management to support accessibility activities
 
 [ ... Tip description goes here ... ]
 
 {:.attach_permalink}
-## Align accessibility rationale with organizational policies
+## Align rationale with organizational policies
 
 [ ... Tip description goes here ... ]
 
 {:.attach_permalink}
-## Develop awareness for accessibility
+## Promote awareness for accessibility
 
 [ ... Tip description goes here ... ]
 
 {:.attach_permalink}
-## Communicate accessibility wins
+## Seek alliances within the organization
 
 [ ... Tip description goes here ... ]
 
 {:.attach_permalink}
-## Create feedback mechanisms
```

Comment:
Feedback to whom? I don't think this is a typical role for advocates. Actually this belongs under managing IMO - it is a process installation rather than advocacy.